### DISCUSSION TITLE
Is meta plugin only available for insiders?

### DISCUSSION THREAD

**Discussion by fredrikaverpil** [squidfunk/mkdocs-material#6021] [Q&A] [answered]
@squidfunk I'm setting up a blog with mkdocs (not insiders) and I was just wondering if the meta plugin (`.meta.yml`) is only available in insiders?

The docs doesn't say anything about this potential limitation:
https://squidfunk.github.io/mkdocs-material/setup/setting-up-a-blog/?h=blog#setting-defaults

I get this on `mkdocs build`:
```
ERROR   -  Config value 'plugins': The "meta" plugin is not installed
```

I'm using the following:
```
mkdocs==1.5.2
mkdocs-material==9.3.1
mkdocs-material-extensions==1.1.1
mkdocs-redirects==1.2.1
mkdocs-rss-plugin==1.8.0
```

**Comment by squidfunk** [comment, marked as answer]
Yes, the [meta plugin](https://squidfunk.github.io/mkdocs-material/plugins/meta/) is [still in Insiders](https://squidfunk.github.io/mkdocs-material/insiders/#16000-chipotle). However, the link to the plugin has gone stale with the latest restructuring of the documentation, so I fixed it in 228599b24 + some other references that needed to be adjusted. After finishing the plugin documentation, we'll be tackling a rewrite of the setup guides, so we'll be moving stuff around in the coming weeks.

**Reply by fredrikaverpil** [reply]
Ah, great - that explains it. Thank you. But then the plan is to leave the meta plugin as part of insiders?

**Reply by squidfunk** [reply]
No, it's going into the community edition! As usual, once [the funding goal](https://squidfunk.github.io/mkdocs-material/insiders/#16000-chipotle) is hit, the plugin goes into the community edition. This is what our [sponsorware strategy](https://squidfunk.github.io/mkdocs-material/insiders/#insiders) is all about: sponsors help drive the project forward with funding and get early access to the latest and greatest features, and the entire community benefits in the long run 🚀 For instance, see the list of [completed goals](https://squidfunk.github.io/mkdocs-material/insiders/#goals-completed) and the associated features to learn which features have been developed with the help of sponsors.

**Reply by fredrikaverpil** [reply]
Aaah, I didn't realize this particular feature was listed in one of the funding stages. I see now it's part of the "$ 16,000 – Chipotle" one. Awesome!

**Reply by pdmosses** [reply]
@squidfunk I think it would be helpful to add the "Insiders" flag at the top of the [`meta` plugin page](https://squidfunk.github.io/mkdocs-material/plugins/meta/#built-in-meta-plugin) .

Referring to it simply as a "built-in" plugin in docs sections _not_ marked as "Insiders" seems inherently confusing. It's only when one scrolls down to the plugin [configuration section](https://squidfunk.github.io/mkdocs-material/plugins/meta/#configuration) that the "insiders" warning appears.

Similarly, the [overview page about plugins](https://squidfunk.github.io/mkdocs-material/plugins/) could include a warning at the top that some (or all?) of the "built-in" plugins are only for insiders. The grid items for the individual plugins there could also indicate which are already available for non-insiders.

Jekyll users will surely expect to be able to configure such "front-matter defaults" without installing any plugin – let alone becoming a sponsor…

**Reply by squidfunk** [reply]
@pdmosses unfortunately, it's not that binary. For example, the social plugin is available in the community edition but also includes Insiders features, as it has been rewritten for Insiders. The same is true for the tags plugin, and the blog plugin. When you come to the section where you learn how to enable each plugin (which you also linked), there's the version identifier + the heart sybmol saying "sponsors only", clearly noting that it's in Insiders.

<img width="743" alt="Bildschirm­foto 2023-10-03 um 17 31 41" src="https://github.com/squidfunk/mkdocs-material/assets/932156/ff4cc5a9-8e9d-45d0-abe5-1898d91389b4">
